release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -1881,6 +1881,12 @@ With this release, replacement control plane nodes are assigned to the correct i
 
 * Previously, the *Samples* page in the {product-title} did not allow distinguishing between the types of samples listed. With this fix, you can identify the sample from the badges displayed on the *Samples* page. (link:https://issues.redhat.com/browse/OCPBUGS-10679[*OCPBUGS-10679*])
 
+[discrete]
+[id="ocp-4-13-documentation-bug-fixes"]
+==== Documentation
+
+Previously, the OpenShift Container Platform documentation included a sub-section titled "Expanding a cluster with on-premise bare metal nodes." However, this has been removed in order to maintain accurate, up-to-date documentation."
```

Comment:
```suggestion
Previously, the OpenShift Container Platform documentation included a sub-section titled "Expanding a cluster with on-premise bare metal nodes." However, this has been removed in order to maintain accurate, up-to-date documentation.
```

Reply:
Removed quotes from original suggestion. Please update and I will remove/add necessary labels.